fondant/dataset.py

```diff
@@ -113,11 +113,13 @@ def _upload_index(self, df: dd.DataFrame):
 
     def _upload_subset(self, name: str, fields: Mapping[str, Field], df: dd.DataFrame):
         # add subset to the manifest
-        manifest_fields = [(field.name, Type[field.type]) for field in fields.values()]
+        manifest_fields = [
+            (field.name, Type[field.type.name]) for field in fields.values()
+        ]
```

Comment:
Had to include this fix to make the component run successfully

Reply:
Had the same issue in my PR